src/tiled/mapdocument.cpp

```diff
@@ -924,6 +924,18 @@ void MapDocument::onLayerRemoved(Layer *layer)
     emit layerRemoved(layer);
 }
 
+void MapDocument::updateAllTemplateInstances()
+{
+    QList<MapObject*> objectList;
+    for (ObjectGroup *group : mMap->objectGroups()) {
+        for (auto object : group->objects()) {
+            object->syncWithTemplate();
+            objectList.append(object);
```

Comment:
Alright I will only update the instances of the changed template instead instead.